Assets/Scripts/Weapons/Projectile.cs

```diff
@@ -1,28 +1,49 @@
 using Mirror;
+using Mirror.Examples.Common.Controllers.Player;
+using Unity.Mathematics;
+using Unity.VisualScripting;
 using UnityEngine;
 using UnityEngine.Tilemaps;
+using UnityEngine.UIElements;
```

Comment:
```suggestion
using UnityEngine;
using UnityEngine.Tilemaps;
```

niepotrzebne importy